addons/binding/org.openhab.binding.solaredge/src/main/java/org/openhab/binding/solaredge/internal/AtomicReferenceUtils.java

```diff
@@ -0,0 +1,43 @@
+/**
+ * Copyright (c) 2010-2018 by the respective copyright holders.
+ *
+ * All rights reserved. This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License v1.0
+ * which accompanies this distribution, and is available at
+ * http://www.eclipse.org/legal/epl-v10.html
+ */
+package org.openhab.binding.solaredge.internal;
+
+import java.util.concurrent.Future;
+import java.util.concurrent.atomic.AtomicReference;
+
+import org.eclipse.jdt.annotation.NonNullByDefault;
+import org.eclipse.jdt.annotation.Nullable;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+/**
+ * utility class which contains useful helper methods. Thus, the interface can be implemented and methods are available
+ * within the class.
+ *
+ * @author Alexander Friese - initial contribution
+ */
+@NonNullByDefault
+public interface AtomicReferenceUtils {
```

Comment:
Utils give me some other ideas this would look more like a `Trait`?

Reply:
It is a collection of utility methods and it is implemented like a trait. But I was not sure if there is a naming convention that suggests using "Trait" in the class name. I could change it to "AtomicReferenceTrait" if you think that it is better understandable.

Reply:
It more clearly expresses its goal, so yes that would be great.